src/app/cards/store/reducers/card.reducers.ts

```diff
@@ -41,6 +42,13 @@ const cardsReducer = createReducer(
       loaded: false,
       errors
     };
+  }),
+  on(cardActions.updateSearch, (state, { search }) => {
+    return {
+      ...state,
+      search,
+      loaded: false
+    };
```

Comment:
Not using this yet, but went ahead and left it in here in case I need it in the future.